Automattic/wp-calypso
# Blog onboarding: Updated redirect link to new launchpad

Related to https://github.com/Automattic/dotcom-forge/issues/2215

## Proposed Changes

* Changed the redirect link for the new launchpad

## Testing Instructions




* Apply this branch on your local Calypso
* Make sure `widgets.wp.com` is sandboxed.
* Create a new site but don't launch it, copy its URL
* Follow the [wpcom-block-editor sync flow](https://github.com/Automattic/wp-calypso/tree/trunk/apps/wpcom-block-editor#dev-workflow). In summary: 
  - cd apps/wpcom-block-editor/src 
  - yarn dev --sync

* Create a new post using the onboarding flag: http://calypso.localhost:3000/post/{domainFromYourNewSite}?showLaunchpad=true
* Inspect the page and open the iFrame URL in a new tab
* After publishing a post, you should get redirected to the new Launchpad `/setup/start-writing/launchpad` instead the old one `/setup/write/launchpad`
* If you have any issues showing the launchpad locally, this might help: p1665762866952539/1665762375.564149-slack-CHN6J22MP
* Try to break the flow and make sure without the flag we won't be redirected
* Run test with this command: `yarn run jest apps/wpcom-block-editor/src/wpcom/features/test/redirect-onboarding-user-after-publishing-post.test.js`


## Pre-merge Checklist
